cpp/include/ray/api/task_caller.h

```diff
@@ -16,11 +16,27 @@ class TaskCaller {
   template <typename... Args>
   ObjectRef<boost::callable_traits::return_type_t<F>> Remote(Args &&... args);
 
+  TaskCaller &SetName(std::string name) {
+    task_options_.name = std::move(name);
+    return *this;
+  }
+
+  TaskCaller &SetResources(std::unordered_map<std::string, double> resources) {
```

Comment:
SetResources -> SetResource

Reply:
We should keep SetResources here, because it will set many resources from a map.